tests/platform_tests/test_memory_exhaustion.py

```diff
@@ -64,6 +64,11 @@ def test_memory_exhaustion(self, duthosts, enum_rand_one_per_hwsku_hostname, loc
         #    background process.
         #  * Some DUTs with few free memory may reboot before ansible receive the result of shell
         #    command, so we add `sleep 5` to ensure ansible receive the result first.
+        # Swapping is turned off so the OOM is triggered in a shorter time.
+
+        res = duthost.command("sudo swapoff -a")
```

Comment:
Is it no-op for the HW which already does not use swapoff

Reply:
yes.